web/utils.go

```diff
@@ -40,7 +40,7 @@ func NewURLTo(appRouter *mux.Router) func(string, ...interface{}) *url.URL {
 	return func(routeName string, ps ...interface{}) *url.URL {
 		route := appRouter.Get(routeName)
 		if route == nil {
-			level.Warn(l).Log("msg", "no such route", "route", routeName, "params", fmt.Sprintf("%v", ps))
+			// level.Warn(l).Log("msg", "no such route", "route", routeName, "params", fmt.Sprintf("%v", ps))
```

Comment:
gotta remember to uncomment this before merging :)

Reply:
If you could mark it with a link to #35 _that would be great_.

Reply:
mark how, by adding a link to #35 in the src?